components/layout/stacked-link.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import { ExternalLinkIcon } from "@heroicons/react/solid";
+import Link from "next/link";
+
+interface StackedListProps {
+  links: [
+    {
+      title: string;
+      url: string;
+    },
+  ];
+}
+
+export default function StackedLink(list: StackedListProps) {
+  const { links } = list;
+  return (
+    <div>
+      {links.map((link, i) => (
+        <Link key={i} href={link.url}>
+          <a target="_blank">
+            <div
+              className={`${
+                i > 0 ? "border-t" : ""
+              }  mx-2 py-4 flex justify-between`}
```

Comment:
Use `clsx` instead of manually concatenating classnames like this.

Reply:
ahh thank you, I just know that. How about this?
![image](https://user-images.githubusercontent.com/3627108/127004919-d6807a13-ef73-44c8-a529-05c67009e413.png)

Reply:
@syauqy empty string -> `undefined` should be ok too.

Reply:
got it, update on the new commit